src/ape/api/query.py

```diff
@@ -69,9 +69,9 @@ class BlockQuery(_BaseBlockQuery):
 
     @classmethod
     def all_fields(cls) -> List[str]:
-        from .providers import BlockAPI
+        from ape_ethereum.ecosystem import Block
 
-        return list(BlockAPI.__fields__)
+        return list(Block.__fields__)
```

Comment:
Keep this the way it was, we should open up a ticket for how to do it better. In general, should not be pulling in a 1st party plugin into ape core.

Reply:
Side note: starting to question whether `columns` is even necessary with the changes to how `QueryAPI` functions

There's no longer really any need to filter out columns anymore with how the iterator works

Reply:
Note: I went with the refactor as described above